spec/factories/orders.rb

```diff
@@ -1,11 +1,27 @@
 FactoryGirl.define do
   factory :order do
-    status "MyString"
-    code "MyString"
-    detail_type "MyString"
-    detail_id 1
-    stockit_contact_id 1
-    stockit_organisation_id 1
-    stockit_id 1
+    status   ["draft", "submitted", "processing", "closed", "cancelled"].sample
+    state   ["draft", "submitted", "processing", "closed", "cancelled"].sample
+
+    code            "#{(65 + Random.rand(26)).chr}#{Random.rand(5)}"
+    detail_type     ["CarryOut", "Shipment", "StockitLocalOrder", "Goodcity"].sample
+
+    description     FFaker::Lorem.sentence
+    purpose_description FFaker::Lorem.sentence
+    stockit_id      1
```

Comment:
Perhaps this can be null rather than 1 as 1 indicates there is a stockit item corresponding to it but really there isn't.